#include "Exception.h"

using namespace std;

int main(int argc, const char *argv[])
{
    throw Exception("hello world");
    return 0;
}
